docs/guides/setup/configuring_data_contexts/how_to_configure_credentials_using_a_yaml_file_or_environment_variables.md

```diff
@@ -61,15 +61,15 @@ Steps
 2. If using a YAML file, set the ``config_variables_file_path`` key in your ``great_expectations.yml`` or leave the default.
 
   ```yaml
-  # great_expectations/great_expectations.yml
+  # great-expectations/great_expectations.yml
```

Comment:
This does not seem to be correct -- should this not remain using the underscore?  If agree, then please verify all similar cases (and must be different from the location on GitHub where the hyphen is used).

Reply:
Ah yes I erroneously changed all instances to underscores and then went back to revise anything that linked to the GE repo. Forgot to account for this let me change it real quick.